transcom/mymove
# B 21093 enforce email domain int

## [B-21093](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1010267)

## Summary

This PR changes the allowed domains for office users to request an account.
Accepted email domains are " .mil, .gov, .edu".

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Go to office login page http://officelocal:3000/sign-in
2. Click "Request Account" on the top right of the screen
3.Fill out information and confirm you can only make an account request if the email domain is in the accepted list.

### Frontend


## Screenshots
_**ANDI Screenshots  for added error messsage**_
<img width="1215" alt="Screenshot 2024-10-01 at 3 20 37 PM" src="https://github.com/user-attachments/assets/eed3df78-5c0a-43b0-88a6-1ad03b6df105">

_**Email Domain error**_
<img width="743" alt="Screenshot 2024-10-01 at 3 19 55 PM" src="https://github.com/user-attachments/assets/a24ba4eb-71ff-4bc0-914c-4412a6e58390">

_**Passing email domain**_
<img width="742" alt="Screenshot 2024-10-01 at 3 20 09 PM" src="https://github.com/user-attachments/assets/80af6e0c-c269-46c9-9af3-65d1cb7f00c3">
